### PR TITLE
Most Recent Update of About me section 

### PR DESCRIPTION
**Pull request by hasenat-eqbali**
Updated Readme file for the recent updates. Added Recent Work and About me Section. Used grid and flex box in CSS styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            width: 150px;
+
+        }
+        .buttom h1{
+    margin: 0 !important;
+}
+.buttom ul{
+    margin: 0 !important;
+}
+    </style>
```
Comment (emmiiorji) [inline index.html]: As a best practice, kindly move inline styles to an external stylesheet to allow for reusability.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="container-list">
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <button class="see-project-btn">See Project</button>
+    </div>
```
Comment (emmiiorji) [inline index.html]: Kindly enclose all of works section within  a `section` tag since they make up a section of the website.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    <div class="project-container">
+        <div class="project">
+            <h1>Profesional Art Printing Data</h1>
```
Comment (emmiiorji) [inline index.html:56]: Please, use any of h2 - h6 here and in other project cards since semantically, a section should not have more than one h1.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @hasenat-eqbali, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

##  Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow
- Good design


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,12 +1,153 @@
...
+            <li><img src="images/twitter.png" alt=""></li>
+            <li><img src="images/medium.png" alt=""></li>
+        </ul><br><br>
+        <img class="corner-img" src="images/Group.png" alt="">
+
+    </section>
+
+    <section class="recent-work-container">
+        <h1>My Recent Works</h1>
+        <hr>
```
Comment (algerina) [inline index.html:39]: - [ Optional] Please consider replacing the h1 with another h tag as you should have no more than one h1 tag.

**Review by algerina — APPROVED** [review]
Hi @hasenat-eqbali   ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
You did great implementing the previews TSE's requirements :+1: 
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**